specification/provisioningservices/resource-manager/readme.md

```diff
@@ -29,15 +27,29 @@ openapi-type: arm
 tag: package-2017-08
 ```
 
+``` yaml
+openapi-type: arm
+tag: package-2017-11
+```
+
 ### Tag: package-2017-08
 
 These settings apply only when `--tag=package-2017-08` is specified on the command line.
 
-``` yaml $(tag) == 'package-2017-08'
+``` yaml $(tag) == 'package-2017-11'
 input-file:
 - Microsoft.Devices/2017-08-21-preview/iotdps.json
```

Comment:
The tag says 2017-11 but the directory is 2017-08-21-preview which I believe is causing the linter failure.